src/js/renderMethod.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable import/no-cycle */
 import { listWrapper } from './elements.js';
```

Comment:
- [ ] Kindly make sure you fix your linters errors instead of disabling them this for best practices reasons and clean code